docs/middlewares.md

```diff
@@ -127,6 +128,40 @@ handler({}, {}, (_, response) => {
 ```
 
 
+## [httpHeaderNormalizer](/src/middlewares/httpHeaderNormalizer.js)
+
+Normalizes HTTP header names to their canonical format. Very useful if clients are
+not using the canonical names of header (e.g. `content-type` as opposed to `Content-Type`).
```

Comment:
Should we mention middleware adds `rawHeaders` property?

Reply:
Very good point, i will do that as well